test_programs/compile_success_empty/comptime_struct/Nargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "comptime_struct"
```

Comment:
We already have a test for structs in comptime_type_definition, from before the type was renamed. Maybe we should move it to `comptime_struct_definition`?

Reply:
Good point. I searched for an existing one but couldn't find it. I'll merge both.